google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -3729,7 +4092,9 @@ TEST(ConnectionImplTest, TransactionOutlivesConnection) {
 
   // Because the transaction outlives the connection, the session is not in
   // the pool when the connection is destroyed, so the session is leaked.
-  EXPECT_CALL(*mock, AsyncDeleteSession).Times(0);
+  // Only the multiplexed session is deleted.
+  EXPECT_CALL(*mock, AsyncDeleteSession(_, _, _, HasSessionName("multiplexed")))
+      .WillOnce(Return(make_ready_future(Status{})));
```

Comment:
Here and elsewhere: did we weak the test?  Should there be something like:

```cc
  EXPECT_CALL(*mock, AsyncDeleteSession(_, _, _, Not(HasSessionName("multiplexed")))).Times(0);
```

??

Reply:
Good catch.  Done.